core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -206,20 +211,30 @@ func (dq *decryptionQueue) SetResult(ciphertextId decryptionPlugin.CiphertextId,
 
 	req, ok := dq.pendingRequests[string(ciphertextId)]
 	if ok {
-		dq.lggr.Debugf("responding with result for pending decryption request ciphertextId %s", string(ciphertextId))
-		req.chPlaintext <- plaintext
+		if err != nil {
+			dq.lggr.Errorw("decryption error", "ciphertextId", toHex(ciphertextId), "error", err)
```

Comment:
Does this mean we will log errors if a user incorrectly encrypts their secrets?

Reply:
Yes. Are you suggesting making it a debug message? You might be right... We're not very consistent with that, for example we log ERROR for invalid CBOR or request-too-big.

I'd agree that it's a better idea to simply log debug messages for all user errors. But maybe let's try to add some extra prom counters so we can notice and investigate those.

Reply:
IMO, we should only log errors for something that is "our" fault, never user error.